### PR TITLE
Remove transform() method on Frustum

### PR DESCRIPTION
**Pull request by nnmm**
It's an unused method that causes the Frustum to be more complex than it needs to be.
And also rename the `query` coordinate system to `world`.

### CODE REVIEW

**Comment by nnmm** [comment]
ptal @feuerste @catevita

File: src/geometry/frustum.rs [hunk 1]
```diff
@@ -12,30 +12,22 @@ use serde::{Deserialize, Serialize};
 /// creating the perspective projection, see also the frustum unit test below.
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct Frustum<S: BaseFloat> {
-    query_from_eye: Isometry3<S>,
-    clip_from_eye: Perspective<S>,
-    query_from_clip: Matrix4<S>,
+    world_from_clip: Matrix4<S>,
```
Comment (feuerste) [inline src/geometry/frustum.rs]: I'm not 100% sure, whether the renaming here makes it clearer, in particular in situations where the Frustum is not defined in world coordinates, but some local coordinates...

Comment (nnmm) [reply src/geometry/frustum.rs]: Ok, I'll undo it

Comment (nnmm) [reply src/geometry/frustum.rs]: Done.

**Pull request merged**